crates/hc_demo_cli/Cargo.toml

```diff
@@ -35,7 +35,7 @@ getrandom = { version = "0.2", features = ["custom"] }
 # Special config for when we're in build_wasm mode
 [target.'cfg(build_wasm)'.build-dependencies]
 flate2 = "1.0.25"
-wasm-opt = "0.112.0"
+wasm-opt = "0.116"
```

Comment:
To test that this worked I had to rebuild the wasms which is why a new version of those has been checked in